### ISSUE TITLE
upload of your addon to blender extensions website

### ISSUE THREAD

**Issue by ghost** [Pullusb/GP_onion_peel#2]
Hi

I want to request the upload of your addon to blender extensions website (Install blender 4.2) so it can be found more easy and become popular in the blender community.

https://extensions.blender.org/

**Comment by Pullusb** [comment]
Hi @blender-girl, thank you for your interest.

Every addon made for grease pencil will have to be rewritten after 4.3 (for Grease Pencil V3) so I prefer to wait for now.

Also, Onion peel specifically might be a bit too hacky to go on official extension platform... And I'd like to limit the addons I upload there. It's always more things to take care of (maintain, document, debug, answer users, etc...).

I see you are posting this message for all addons you like 😄 (just answered here https://github.com/Pullusb/GP_refine_strokes/issues/4)

Now I'm curious. Can you post a list of repository links to the other addons where you suggested this ?
Seeing you know of `Onion peel` and `Refine strokes`, I'd like to see what other tools you're using ^^

**Comment by ghost** [comment]
I use all of these and asked them too 🥰

https://lacuisine.tech/which-add-ons-to-start-with-grease-pencil/

**Comment by Pullusb** [comment]
Ah yes, that's a good resource ;)

**Comment by Pullusb** [comment]
Hi again @blender-girl,
just to be clear (Because I realized it's not.)

You initially just asked if I, or any other devs you contacted, would be willing to upload our addons to the extension platform ?
Not as a request to do it yourself, right ?
Because there is no way the dev would let someone else handle their work.

Also, I hope you did not flood too many people with those requests.... Add-ons developers are well aware of the extension platform, and if they are willing to upload their work, they will do it themselves in due time.

Cheers

**Comment by ghost** [comment]
> Not as a request to do it yourself, right ?
> Because there is no way the dev would let someone else handle their work.

no, not myself. I'm not even a developer :) I believe it's also a rule by the blender foundation ONLY the developer/maintainer can upload the code.

I just strongly believe add-ons should be on the extension website because otherwise many add-ons stay in obscurity a bit.
I discovered your add-ons by luck when visiting the link above.
